torch/onnx/symbolic_opset9.py

```diff
@@ -5006,7 +5272,9 @@ def tolist(g, input, dim_val, elem_ty_val):
     # Symbolic functions that need extra context
     # -----------------------------------------------------------------------------
     @staticmethod
-    def device(ctx: torch.onnx.SymbolicContext, g: _C.Graph, *inputs, **kwargs) -> None:
+    def device(
+        ctx: _exporter_states.SymbolicContext, g: _C.Graph, *inputs, **kwargs
```

Comment:
To clarify, is `torch.onnx.SymbolicContext` still supported syntax?

I feel the previous version is better. Using class inside private module seems strange for users especially when writing custom symbolic functions on their own. The rest of changes using new `module_name.class_name` format looks good.

Reply:
Users will still use `torch.onnx.SymbolicContext`. Internally we use `_exporter_states.SymbolicContext` to avoid references to torch.onnx. (for a clean import dependency)

Reply:
Would it be clearer to special case this usage with `from _exporter_states import SymbolicContext`, then annotate as `SymbolicContext`, what do you think?

Reply:
Sure! We can do that. We just need to make sure `_exporter_states` doesn't take many dependencies to complicate the dependency graph (which is the case now so we are good)

imo we should treat it as exceptional and not use it as an example for future usages

Reply:
Done